GitUI/UserControls/FileStatusList.cs

```diff
@@ -800,24 +800,12 @@ public async Task OpenSubmoduleAsync()
             var submoduleName = SelectedItem.Item.Name;
 
             var status = await SelectedItem.Item.GetSubmoduleStatusAsync().ConfigureAwait(false);
-            var selected = SelectedItem.SecondRevision.ObjectId == ObjectId.WorkTreeId
-                ? SelectedItem.SecondRevision.ObjectId.ToString()
-                : status?.Commit.ToString() ?? string.Empty;
-            var first = string.IsNullOrWhiteSpace(status?.OldCommit?.ToString())
-                ? string.Empty
-                : $",{status.OldCommit}";
+            ObjectId selectedId = SelectedItem.SecondRevision?.ObjectId == ObjectId.WorkTreeId
+                ? SelectedItem.SecondRevision.ObjectId
```

Comment:
```suggestion
                ? ObjectId.WorkTreeId
```
(yes this is following what I wrote,...)

Reply:
done